Sources/MongoSwift/MongoCollection+BulkWrite.swift

```diff
@@ -0,0 +1,131 @@
+import libmongoc
+
+/// An extension of `MongoCollection` encapsulating bulk write operations.
+extension MongoCollection {
+    /**
+     * Sends a batch of writes to the server at the same time.
+     *
+     * - Parameters:
+     *   - requests: an `[WriteModel]` containing the writes to perform
```

Comment:
I think "a" is fine here, unless you were actually writing "an array of `WriteModel`s".